pages/c/_cluster/explorer/index.vue

```diff
@@ -158,7 +160,15 @@ export default {
     },
 
     accessibleResources() {
-      return RESOURCES.filter(resource => this.$store.getters['cluster/schemaFor'](resource));
+      // This is a list of IDs for allowed resources counts.
+      const defaultAllowedResources = Object.keys(this.clusterCounts?.[0].counts).filter((typeId) => {
+        return this.$store.getters['type-map/isIgnored']({ id: typeId });
+      });
+
+      // Merge with RESOURCES list
+      const allowedResources = [...new Set([...defaultAllowedResources, ...RESOURCES])];
```

Comment:
Do we need this?

Reply:
Yes, 
e.g. `NAMESPACES = 'namespaces` is initially included as a resource. However, it's also an `ignoreType`

https://github.com/rancher/dashboard/blob/74948e156e1f5606e1037105261afc95838c1e6b/pages/c/_cluster/explorer/index.vue#L43

and 

https://github.com/rancher/dashboard/blob/74948e156e1f5606e1037105261afc95838c1e6b/config/product/explorer.js#L98